docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -16,21 +16,24 @@ The User Feedback Widget offers many customization options, and if the available
-| ----------------- | ------------------------ | ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| `showName`        | `boolean`                | `true`                                | Displays the name field on the Feedback form, however will still capture the name (if available) from Sentry SDK context.   |
-| `showEmail`       | `boolean`                | `true`                                | Displays the email field on the Feedback form, however will still capture the email (if available) from Sentry SDK context. |
-| `isNameRequired`  | `boolean`                | `false`                               | Requires the name field on the Feedback form to be filled in.                                                               |
-| `isEmailRequired` | `boolean`                | `false`                               | Requires the email field on the Feedback form to be filled in.                                                              |
-| `useSentryUser`   | `Record<string, string>` | `{ email: 'email', name: 'username'}` | Map of the `email` and `name` fields to the corresponding Sentry SDK user fields that were called with `Sentry.setUser`.    |
+The following options can be configured for the integration in `feedbackIntegration({})`:
+
+| key               | type                     | default                               | description                                                                                                                    |
+| ----------------- | ------------------------ | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `showName`        | `boolean`                | `true`                                | Displays the name field on the Feedback form, however will still capture the name (if available) from Sentry SDK context.      |
+| `showEmail`       | `boolean`                | `true`                                | Displays the email field on the Feedback form, however will still capture the email (if available) from Sentry SDK context.    |
+| `isNameRequired`  | `boolean`                | `false`                               | Requires the name field on the Feedback form to be filled in.                                                                  |
+| `isEmailRequired` | `boolean`                | `false`                               | Requires the email field on the Feedback form to be filled in.                                                                 |
+| `useSentryUser`   | `Record<string, string>` | `{ email: 'email', name: 'username'}` | Map of the `email` and `name` fields to the corresponding Sentry SDK user fields that were called with `Sentry.setUser`.       |
```

Comment:
```suggestion
| `useSentryUser`   | `Record<string, string>` | `{ email: 'email', name: 'username'}` | Sets the `email` and `name` fields to the corresponding Sentry SDK user fields that were called with `Sentry.setUser`.       |
```
Is this more accurate? If the user inputs a different name or email in the form, will the SentryUser fields override it?